docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -2,16 +2,16 @@
 Submitting contributions
 ========================
 
-We're always grateful for contributions to Django's code. Indeed, bug reports
-with associated contributions will get fixed *far* more quickly than those
-without a solution.
+We are always grateful for patches to Django's code.
+To ensure a smooth process, please create a ticket on our `ticket tracker`_. before submitting your patch.
+Bug reports with associated tickets and contributions will be fixed *far* more quickly than those without.
 
 Typo fixes and trivial documentation changes
 ============================================
 
-If you are fixing a really trivial issue, for example changing a word in the
-documentation, the preferred way to provide the patch is using GitHub pull
-requests without a Trac ticket.
+If you are fixing a trivial issue, for example changing a word in the
+documentation, the preferred process is to create a Trac ticket first.
+Once the ticket is accepted, you can then provide the patch using a GitHub pull request.
```

Comment:
I think this part should largely stay as it was: I don't think we want to encourage tickets for typo fixes. Maybe we just need to clarify what kind of wording changes we mean. I just adjusted the wording of an error message in #18338, and I had a ticket because it was user-facing and changed the sense of the message. But if my change had only been to fix a typo, a ticket would have been unnecessary. I took the point of this section to be about _typo-like_ changes to docs that are uncontroversial in any possible way.